app/lib/brexit_checker/account_jwt.rb

```diff
@@ -0,0 +1,33 @@
+class BrexitChecker::AccountJwt
+  def initialize(criteria_keys)
+    @criteria_keys = criteria_keys
+  end
+
+  def encode(key = ecdsa_key, algorithmn = "ES256")
+    JWT.encode payload, key, algorithmn
+  end
+
+private
+
+  attr_reader :criteria_keys
+
+  def payload
+    {
+      uid: uid,
+      data: criteria_keys,
+      scopes: scopes,
+    }
+  end
+
+  def uid
+    ENV.fetch("GOVUK_ACCOUNT_OAUTH_CLIENT_ID")
+  end
+
+  def scopes
+    "update-transition-checker-results"
```

Comment:
The scope is transition_checker and the attribute is transition_checker_state: https://github.com/alphagov/govuk-attribute-service-prototype/blob/main/config/scopes.yml#L8

Reply:
[amended here](https://github.com/alphagov/finder-frontend/pull/2207/commits/e43d83b52b5813a84303a4ecc93f047fc719a2ad#diff-e4257d6c3bb3ff33ad32620a2d04b871R17) and removed the scopes method